tensor2tensor/data_generators/problem.py

```diff
@@ -933,6 +933,15 @@ def prepare_for_output(example):
 
     return dataset
 
+  @property
+  def export_asserts(self):
```

Comment:
Is this function strictly needed, or are you using it for some private problems? It's fine with me, but I'm worried that without any tests it'll just get deleted in the next code cleanup...

Also: the name should be "assets" and not "asserts", right?

Reply:
Yeah we're using it in some yet unreleased code that uses t2t with our own problems.

And yes thanks for spotting the typo, I'll fix it.